test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
Uh... I would expect this triple to be *dynamically* added by the glob code, not *statically* added to the `sampleContainer/example1.ttl` graph.